src/custom/utils/operator/index.ts

```diff
@@ -153,20 +154,32 @@ function toApiAddress(address: string, chainId: ChainId): string {
 
   return address
 }
-
-async function _getJson(chainId: ChainId, url: string): Promise<any> {
+type RequestType = 'Order' | 'Quote'
+async function _getJson(chainId: ChainId, url: string, type: RequestType): Promise<any> {
```

Comment:
I don't like this request type.
As I understand, this is only for error handling, correct?
In that case, it shouldn't be a parameter. If there's any translation between api error type and our internal error type it should be inside each method.

For reference, I've refactored how this is done in the cancellation PR and the logic for dealing with this is inside each individual function https://github.com/gnosis/cowswap/blob/995479cff0e7ac56c4a06bca525916cecbaabe05/src/custom/utils/operator/index.ts#L86-L105

Reply:
looking 👁️ 👄 👁️

Reply:
changes inside #770